### ISSUE TITLE
ATOM feed for a Jeykll Blog

### ISSUE THREAD

**Issue by MaryBethBaker** [canada-ca/canada-ca.github.io#4]
Quick question for @smellems: Does this come with an ATOM feed? Or is this a plugin we'll need to install? I checked here https://jekyllrb.com/docs/plugins/ 
And found 1 options for RSS: 
• RssGenerator by Assaf Gelber: Automatically creates an RSS 2.0 feed from your posts.
But the GC standard is ATOM... Is this an ATOM feed tool or something else? 
* markdown-writer: An Atom package for Jekyll. It can create new posts/drafts, manage tags/categories, insert link/images and add many useful key mappings.

**Comment by sboots** [comment]
(Just in case it's helpful! There's a `jekyll-feed` plugin that provides an Atom feed, and it's compatible with or built into Github Pages. That's what we're using by accident on digital.canada.ca.) :)

**Comment by MaryBethBaker** [comment]
Awesome - we've setup a Jekyll site but I didn't know about this functionality. That's why I asked your team too on your account.

**Comment by sboots** [comment]
Awesome! 👍 

It looks like
```
gems:
  - jekyll-feed
```

in our `_config.yml`.

**Comment by smellems** [comment]
@MaryBethBaker @sboots I was able to take a look at the Jekyll setup for the digital.canada.ca.  They have two seperate repos for English and French.  For the Canada-ca I used just one repo for both.  That means, I think, that enabling jekyll-feed would not look at French and English content and just show everything.

But good news!  You already have an RSS feed available.  There's a link to it on the home page.
https://canada-ca.github.io/feed.xml
and in French
https://canada-ca.github.io/flux.xml

I don't know if this a good way of doing it, that's how it was when I forked it..
